karyon2-examples/src/main/java/netflix/karyon/examples/hellonoss/server/rxnetty/MyApplicationRunner.java

```diff
@@ -14,17 +17,26 @@
  */
 public class MyApplicationRunner {
 
+    public static class HealthCheckStrategyModule extends AbstractModule {
+        @Override
+        protected void configure() {
+            bind(HealthCheckInvocationStrategy.class).to(SyncHealthCheckInvocationStrategy.class);
+        }
+    }
+
     public static void main(String[] args) {
         HealthCheck healthCheckHandler = new HealthCheck();
         Karyon.forRequestHandler(8888,
-                                 new RxNettyHandler("/healthcheck",
-                                                    new HealthCheckEndpoint(healthCheckHandler)),
-                                 new KaryonBootstrapModule(healthCheckHandler),
-                                 new ArchaiusBootstrapModule("hello-netflix-oss"),
-                                 // KaryonEurekaModule.asBootstrapModule(), /* Uncomment if you need eureka */
-                                 KaryonWebAdminModule.asBootstrapModule(),
-                                 ShutdownModule.asBootstrapModule(),
-                                 KaryonServoModule.asBootstrapModule())
-              .startAndWaitTillShutdown();
+                new RxNettyHandler("/healthcheck",
+                        new HealthCheckEndpoint(healthCheckHandler)),
+                new KaryonBootstrapModule(healthCheckHandler),
+                new ArchaiusBootstrapModule("hello-netflix-oss"),
+                // KaryonEurekaModule.asBootstrapModule(), /* Uncomment if you need eureka */
+                Karyon.toBootstrapModule(KaryonWebAdminModule.class),
+                ShutdownModule.asBootstrapModule(),
+                KaryonServoModule.asBootstrapModule(),
+                Karyon.toBootstrapModule(HealthCheckStrategyModule.class)
```

Comment:
I think we should not encourage people to use the admin healthcheck strategy module as it does not give a comprehensive indication of the health of the application.